iree/compiler/Dialect/HAL/IR/LoweringConfig.td

```diff
@@ -46,6 +46,8 @@ def TileSizesListAttr :
 def TranslationInfoAttr :
   StructAttr<"TranslationInfo", HAL_Dialect, [
     StructFieldAttr<"passPipeline", DispatchLoweringPassPipelineEnum>,
+    StructFieldAttr<"fullWorkload",
```

Comment:
I am not sure this is the right place for it. Essentially the `flow.workgroups.dispatch` operation already has this information. What we need is to translate that through to the `hal.executable` operation "somehow". Its a bit tricky cause its an SSA value. Adding this here would imply that we need this to be consistent with what is calculated at the flow level. We already have this issue, where we are trying to get the workload size from the problem size. That is something we need to fix too, but I dont think we want to add another way of carrying the `workload` information. (One idea there is to have a `hal.executable.workload` operation which gets resolved to the workload value if we really needed, still that actually does not address the issue here).

For now since this seems to be only an issue with convolution, I would suggest that you just use `getUntiledResultShape` to get the workload size.

FYI long term solution: What we really want is that the sizes of the "tiles" need to be static when the tile size divides the problem size. All the `affine.min` canonicalizations are a red-herring. We just need to replace

```
%actual_ts = affine_min(...)
```

with 

```
%0 = remi %ub, %ts
%cond = cmpi, "eq", %0, %c0
%partial_ts = affine_min(...)`
%actual_ts = select %cond, %ts, %partial_ts
```

With this for the case where the tile size divides the problem size, canonicalization will clear all the `affine_min`s away. I am just waiting for this branch to be merged. After that we can probably get rid of the complex affine min canonicalization patterns.